core/gather/gatherers/bf-cache-failures.js

```diff
@@ -0,0 +1,158 @@
+
+class BFCacheFailures extends FRGatherer {
+  /** @type {LH.Gatherer.GathererMeta<'DevtoolsLog'>} */
+  meta = {
+    supportedModes: ['navigation', 'timespan'],
+    dependencies: {DevtoolsLog: DevtoolsLog.symbol},
+  };
+
+  /**
+   * @param {LH.Crdp.Page.BackForwardCacheNotRestoredExplanation[]} errorList
+   * @return {LH.Artifacts.BFCacheFailure}
+   */
+  static processBFCacheEventList(errorList) {
+    /** @type {LH.Artifacts.BFCacheNotRestoredReasonsTree} */
+    const notRestoredReasonsTree = {
+      Circumstantial: {},
+      PageSupportNeeded: {},
+      SupportPending: {},
+    };
+
+    for (const err of errorList) {
+      const bfCacheErrorsMap = notRestoredReasonsTree[err.type];
+      bfCacheErrorsMap[err.reason] = [];
+    }
+
+    return {notRestoredReasonsTree};
+  }
+
+  /**
+   * @param {LH.Crdp.Page.BackForwardCacheNotRestoredExplanationTree} errorTree
+   * @return {LH.Artifacts.BFCacheFailure}
+   */
+  static processBFCacheEventTree(errorTree) {
+    /** @type {LH.Artifacts.BFCacheNotRestoredReasonsTree} */
+    const notRestoredReasonsTree = {
+      Circumstantial: {},
+      PageSupportNeeded: {},
+      SupportPending: {},
+    };
+
+    /**
+     * @param {LH.Crdp.Page.BackForwardCacheNotRestoredExplanationTree} node
+     */
+    function traverse(node) {
+      for (const error of node.explanations) {
+        const bfCacheErrorsMap = notRestoredReasonsTree[error.type];
+        const frameUrls = bfCacheErrorsMap[error.reason] || [];
+        frameUrls.push(node.url);
+        bfCacheErrorsMap[error.reason] = frameUrls;
+      }
+
+      for (const child of node.children) {
+        traverse(child);
+      }
+    }
+
+    traverse(errorTree);
+
+    return {notRestoredReasonsTree};
+  }
+
+  /**
+   * @param {LH.Crdp.Page.BackForwardCacheNotUsedEvent|undefined} event
+   * @return {LH.Artifacts.BFCacheFailure}
+   */
+  static processBFCacheEvent(event) {
+    if (event?.notRestoredExplanationsTree) {
+      return BFCacheFailures.processBFCacheEventTree(event.notRestoredExplanationsTree);
+    }
+    return BFCacheFailures.processBFCacheEventList(event?.notRestoredExplanations || []);
+  }
+
+  /**
+   * @param {LH.Gatherer.FRTransitionalContext} context
+   * @return {Promise<LH.Crdp.Page.BackForwardCacheNotUsedEvent|undefined>}
+   */
+  async activelyCollectBFCacheEvent(context) {
+    const session = context.driver.defaultSession;
+
+    /** @type {LH.Crdp.Page.BackForwardCacheNotUsedEvent|undefined} */
+    let bfCacheEvent = undefined;
+
+    /**
+     * @param {LH.Crdp.Page.BackForwardCacheNotUsedEvent} event
+     */
+    function onBfCacheNotUsed(event) {
+      bfCacheEvent = event;
+    }
+
+    session.on('Page.backForwardCacheNotUsed', onBfCacheNotUsed);
+
+    const history = await session.sendCommand('Page.getNavigationHistory');
+    const entry = history.entries[history.currentIndex];
+
+    await Promise.all([
+      session.sendCommand('Page.navigate', {url: 'chrome://terms'}),
```

Comment:
Looks like we can use `about:blank` here.

Another option is to pull in `context.settings.blankPage` so that it's customizable with the config.

Reply:
Let's keep to about:blank - it's a valid page for all chromium browsers, and there's no reason to use anything else so best not to conflate with that option (which could result in subtle breakage here)